### PR TITLE
:rocket: Adding a logout option for the website

### LINKED ISSUE #71: 🚀  Adding a Logout Option
In order to make the website usable to different users we'll need to add a logout option.
Make sure to connect the logout link in the base_template to the correct url.

### PR DESCRIPTION
**Pull request by amittcohen**
Added a logout option, by doing the following steps:
- Updated the urls.py file in the login app to contain a logout option
- Configured a LOGOUT_REDIRECT_URL in the settings.py file
- Updated the logout button link in the base template
- Removed unecessary functions in the views.py file in the login app

closes #71

### CODE REVIEW

**Comment by paOmer** [comment]
Hi @amittcohen,
I think that removing the login `views.py` file content should be moved to its own PR.

Also, I think it would be clearer to the user, that after login out he would get redirected to one of this 2 options:
- To the main feed
- To a logout template which says to the user he successfully logged out, and there you could add a link to the login page.

**Comment by paOmer** [comment]
Hi @amittcohen,
I think that removing the login `views.py` file content should be moved to its own PR.

Also, I think it would be clearer to the user, that after login out he would get redirected to one of this 2 options:
- To the main feed
- To a logout template which says to the user he successfully logged out, and there you could add a link to the login page.

**Comment by amittcohen** [comment]
> Hi @amittcohen, I think that removing the login `views.py` file content should be moved to its own PR.
> 
> Also, I think it would be clearer to the user, that after login out he would get redirected to one of this 2 options:
> 
> * To the main feed
> * To a logout template which says to the user he successfully logged out, and there you could add a link to the login page.

Regarding the views.py file, I have created another pull request (#116 )

**Review by Yarboa — CHANGES_REQUESTED** [review]
Please add tests,
You could find samples at the Appendix of UI talk
Only login tests i see are those:
https://github.com/redhat-beyond/JobSeeker/blob/main/login/tests.py#L1

There few tests here to verify the login work

File: app/settings.py [hunk 1]
```diff
@@ -138,3 +138,5 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'personal_profile/static/personal_profile/images')
 
 LOGIN_REDIRECT_URL = '/about/'
+
+LOGOUT_REDIRECT_URL = '/login/'
```
Comment (paOmer) [inline app/settings.py:142]: I suggest to change it to the feed page url: `/`

Also rebase this PR

**Review by paOmer — CHANGES_REQUESTED** [review]
Add a test that verify a user gets logged out when trying the logout link

**Comment by amittcohen** [comment]
@paOmer @Yarboa Added the test

**Review by paOmer — APPROVED** [review]
LGTM

**Review by DeanBiton — APPROVED** [review]

**Review by Yarboa — APPROVED** [review]

**Pull request merged**